src/redux/missions/missions.css

```diff
@@ -0,0 +1,103 @@
+.table-container {
+  width: 90%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin: 0 auto;
+  margin-top: 24px;
+  overflow-x: auto;
+}
+
+.mission-table {
+  border-collapse: collapse;
+  width: 100%;
+}
+
+.mission-table td,
+.mission-table th,
+.mission-table {
+  border: 1px solid #ddd;
+  padding: 8px;
+}
+
+.mission-table th {
+  padding-top: 12px;
```

Comment:
- [ OPTIONAL] 👋 Hey there! Great job on your PR! 🎉 Just one friendly suggestion: let's keep our project organized. Please avoid putting CSS files in the Redux folder. Instead, consider creating a separate "styles" or "css" folder for your stylesheets. This helps keep our codebase clean and makes it easier for everyone to find what they need. Thanks for your hard work, and keep up the good coding! 😊🚀 #KeepItTidy